src/sgame/sg_maprotation.cpp

```diff
@@ -1042,6 +1042,14 @@ Send commands to the server to actually change the map
 static void G_IssueMapChange( int index, int rotation )
 {
 	mrNode_t *node = mapRotations.rotations[ rotation ].nodes[ index ];
+  mrCondition_t *condition;
```

Comment:
Prefer to declare variables in the narrowest possible scope (inside the `if` block in this case)